package.json

```diff
@@ -14,11 +14,7 @@
     "report:bundle-size:time-series": "node ./scripts/reports/bundle-size/time-series.js",
     "cypress:test": "lerna run cypress:test",
     "commit": "git-cz",
-    "bump:version:pre": "node ./scripts/deploy/bump-version.js",
-    "bump:version:graduate": "node ./scripts/deploy/bump-version.js --graduate",
-    "version": "lerna version --conventional-commits",
-    "version:pre": "lerna version --conventional-commits --conventional-prerelease",
-    "version:graduate": "lerna version --conventional-commits --conventional-graduate",
```

Comment:
We no longer need two `bump:version` scripts.

I didn't find any instance of `version` being used.

As for `version:pre` and `version:graduate`, I find it less confusing if they are removed and the script uses `npx` instead.

Reply:
I don't know what a graduate version refers too. Is it a future version or the most recently updated version?

Reply:
If the current version is `v1.1.1`, and between the current & last graduate there was:
* A `feat` commit, with any amount of other commits
  * Then the current `graduate` will upgrade the version to `v1.2.0`
* Any amount of commits, but no `feat` commit
  * Then the current `graduate` will upgrade the version to `v1.1.2`